server/auth.js

```diff
@@ -25,25 +25,10 @@ const createJWTBearerToken = user => jwt.sign({}, RSA_PRIVATE_KEY, {
 // To protect routes
 const checkIfAuthenticated = expressJwt({
   secret: RSA_PUBLIC_KEY,
+  // comment out following line if sending Postman requests as cookie is retrieved differently
   getToken: req => req.cookies.SESSIONID,
 }).unless({ path: ['/', '/chefauth', '/userauth'] });
 
-const redirect = (req, res, next) => {
```

Comment:
Did you take this out completely or just move it?

Reply:
Took it out completely. Two auth functions we didn't need (they weren't really working anyway). The redirect happens elsewhere.